pom.xml

```diff
@@ -63,7 +62,6 @@
 							<vmArgs>
 								<vmArg>-Djavax.xml.accessExternalSchema=all</vmArg>
 							</vmArgs>
-							<extension>true</extension>
```

Comment:
@joaofreitas-qubit why was this dropped?

Reply:
You just solved the mistery in the santander-sdk, the module was not compiling because i mistankenly removed this